java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/user/MapApiController.java

```diff
@@ -379,6 +379,10 @@ public ResponseEntity<String> listFiles(@RequestParam(required = false) String n
 						GpxFile gpxFile;
 						try (Source source = new Buffer().readFrom(in)) {
 							gpxFile = GpxUtilities.INSTANCE.loadGpxFile(source);
+						} catch (IOException e) {
+							LOG.error("Error reading gpx file: " + uf.name + " (id=" + uf.id + ") (userid=" + uf.userid + ")");
```

Comment:
LOG.error("web-list-files: loadGpxFile error: " + uf.name + " (id=" + uf.id + ") (userid=" + uf.userid + ")");

please use similar keywords in message to make it "greppable" easier in the logs (grep "web-list-files" ...)